core/src/test/java/org/opensearch/sql/analysis/AnalyzerTest.java

```diff
@@ -694,7 +712,7 @@ public void parse_relation() {
             AstDSL.parse(
                 AstDSL.relation("schema"),
                 AstDSL.field("string_value"),
-                AstDSL.stringLiteral("(?<group>.*)")),
+                stringLiteral("(?<group>.*)")),
```

Comment:
Why?

Reply:
I believe this was an accidental change done in a previous draft PR. Should be fixed under [57ec0691b36f1e0010c034df1a59ef6b364feaf4](https://github.com/Bit-Quill/opensearch-project-sql/pull/96/commits/57ec0691b36f1e0010c034df1a59ef6b364feaf4).